js/core/tests/flow_test.ts

```diff
@@ -52,33 +53,35 @@ function createTestStreamingFlow(ai: Genkit) {
 }
 
 describe('flow', () => {
-  let ai: Genkit;
+  let registry: Registry;
 
   beforeEach(() => {
     // Skips starting reflection server.
     delete process.env.GENKIT_ENV;
-    ai = genkit({});
+    registry = new Registry();
   });
 
   describe('runFlow', () => {
     it('should run the flow', async () => {
-      const testFlow = createTestFlow(ai);
+      const testFlow = runWithRegistry(registry, createTestFlow);
```

Comment:
Why go back to runWithRegistry?

Reply:
Separation of concerns. `Genkit` is outside of this package and all of the primitives operate on the registry alone. Makes more sense to test them in isolation.

Reply:
Makes sense! I have some ideas around that... I'd like `runWithRegistry` to only be used for backwards compatibility in plugins. Don't worry about it for now. Future cleanup.